backend/tournesol/views/preview.py

```diff
@@ -8,12 +8,64 @@
 from PIL import Image, ImageDraw, ImageFont
 from rest_framework.views import APIView
 
+from settings.settings import BASE_DIR
+from tournesol.entities.video import TYPE_VIDEO
 from tournesol.models.entity import Entity
 from tournesol.utils.cache import cache_page_no_i18n
 
 logger = logging.getLogger(__name__)
 
 
+FOOTER_FONT_LOCATION = "tournesol/resources/Poppins-Medium.ttf"
+ENTITY_N_COMPARISONS_XY = (226, 26)
+ENTITY_N_CONTRIBUTORS_YX = (226, 40)
+ENTITY_TITLE_XY = (10, 6)
+TOURNESOL_SCORE_XY = (10, 26)
+
+
+def get_preview_font_config():
+    fnt = ImageFont.truetype(str(BASE_DIR / FOOTER_FONT_LOCATION), 20)
+    fnt_title = ImageFont.truetype(str(BASE_DIR / FOOTER_FONT_LOCATION), 14)
+    fnt_ratings = ImageFont.truetype(str(BASE_DIR / FOOTER_FONT_LOCATION), 12)
+    return fnt, fnt_title, fnt_ratings
+
+
+def get_preview_footer(entity, fnt, fnt_title, fnt_ratings) -> Image:
+    tournesol_footer = Image.new("RGBA", (320, 60), (255, 200, 0, 255))
+    tournesol_footer_draw = ImageDraw.Draw(tournesol_footer)
+
+    truncated_title = entity.metadata.get("name", "")[:200]
+    # TODO: optimize this with a dichotomic search
+    while tournesol_footer_draw.textlength(truncated_title, font=fnt_title) > 300:
+        truncated_title = truncated_title[:-4] + "..."
+    tournesol_footer_draw.text(
+        ENTITY_TITLE_XY, truncated_title, font=fnt_title, fill=(29, 26, 20, 255)
+    )
+
+    score = entity.tournesol_score
+    if score is not None:
+        tournesol_footer_draw.text(
+            TOURNESOL_SCORE_XY,
+            "TS %.0f" % score,
+            font=fnt,
+            fill=(29, 26, 20, 255),
+        )
+
+        tournesol_footer_draw.text(
+            ENTITY_N_COMPARISONS_XY,
+            f"{entity.rating_n_ratings} comparisons",
+            font=fnt_ratings,
+            fill=(29, 26, 20, 255),
+        )
+        tournesol_footer_draw.text(
+            ENTITY_N_CONTRIBUTORS_YX,
+            f"{entity.rating_n_contributors} contributors",
+            font=fnt_ratings,
+            fill=(29, 26, 20, 255),
+        )
```

Comment:
We could split the number and text such that it doesn't get misaligned when the numbers don't have the same digits. This could be done with a right anchor on the count and left anchor on the word 

https://pillow.readthedocs.io/en/stable/handbook/text-anchors.html